src/Modules/Visualization/ShowField.cc

```diff
@@ -100,6 +100,21 @@ void ShowFieldModule::execute()
 {
   boost::shared_ptr<SCIRun::Field> field = getRequiredInput(Field);
   boost::optional<boost::shared_ptr<SCIRun::Core::Datatypes::ColorMap>> colorMap = getOptionalInput(ColorMapObject);
+  
+  //set the default colormap scaling to be full range if no rescaling was provided.
+  if (colorMap &&
```

Comment:
@aylakhan @brigb123 @cbrightsci I'm not sure this code should be in ShowField. It kind of defeats the purpose of the RescaleColorMap module, and just moves the old ShowField color scaling to a different place. Plus it will introduce a bug with tensor field visualization.